cmd/merge.go

```diff
@@ -37,7 +37,7 @@ func mergeExec(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if err := merge.MergeOpenAPIDocuments(cmd.Context(), inSchemas, outFile, "", ""); err != nil {
+	if err := merge.MergeOpenAPIDocuments(cmd.Context(), inSchemas, outFile, "speakeasy-recommended", ""); err != nil {
```

Comment:
should validation infect the various document operations (in this case merging)?

Reply:
so this is just restoring the original behaviour which was meant to be using this but it got set to an empty string in the last integration I did by mistake